prow/knative/plugins.yaml

```diff
@@ -16,6 +16,7 @@
 approve:
 - repos:
   - "knative"
+  # TODO(mattmoor): Should we disable this for sandbox?
```

Comment:
@chaodaiG @chizhg Please advise on whether I should drop this?

Reply:
Not sure about the motivation. It reacts to `/lgtm`(When lgtm as approve is enabled) and `/approve` comments and approves PR. Keeping this on shouldn't hurt imo

Reply:
Well, without OWNERS files it leaves a confusing "NOT APPROVED" message file before the PR is merged anyhow. 😅

Reply:
See: https://github.com/knative-sandbox/net-contour/pull/434#issuecomment-782245261

Reply:
ah. I see 😅

Reply:
If there are still repos don't have OWNERS file, maybe should drop it for now?

Reply:
The goal would be to drop OWNERS from all repos, right now the CODEOWNERS files coexist, but with this PR the source of truth will cut over to CODEOWNERS as the source of truth and OWNERS will become meaningless.